system/vrops-custom-importer/values.yaml

```diff
@@ -0,0 +1,20 @@
+owner-info:
+  support-group: compute
+  maintainers:
+    - Christoph Richter
+  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/system/vrops-custom-importer
+name: vrops-custom-importer
+namespace: vrops-custom-importer
+schedule: "*/5 * * * *"
+image:
+  repository:
+  tag: latest
```

Comment:
For better backward compatibility I'd use a specific version which can be bumped. Otherwise it might change at pod kick time.